### DISCUSSION TITLE
2006/11/15/%E5%9C%A8%E7%A8%8B%E5%BA%8F%E4%B8%AD%E4%BD%BF%E7%94%A8%E7%B1%BB%E5%9E%8B%E5%AE%9A%E4%B9%89%EF%BC%88type-def%EF%BC%89

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#129] [Announcements]
# 2006/11/15/%E5%9C%A8%E7%A8%8B%E5%BA%8F%E4%B8%AD%E4%BD%BF%E7%94%A8%E7%B1%BB%E5%9E%8B%E5%AE%9A%E4%B9%89%EF%BC%88type-def%EF%BC%89

在 C 和 C++ 中，程序员可以使用 typedef 来定义自己的数据类型。typedef 的首要好处是使得程序具有更好的可读性，可以为某种数据类型起一个针对你的程序有意义的名字；另一个好处就是，如果这个数据类型需要作改动，只要在 typedef 语句处改动就可以了，程序中使用了这个数据类型的代码部分不需要作任何改动。

https://blog.qizhen.xyz/2006/11/15/%E5%9C%A8%E7%A8%8B%E5%BA%8F%E4%B8%AD%E4%BD%BF%E7%94%A8%E7%B1%BB%E5%9E%8B%E5%AE%9A%E4%B9%89%EF%BC%88type-def%EF%BC%89

**Comment by ruanqizhen** [comment]
之前博客下的留言：


K
2007年02月21日 21:00 编辑
技术的东西是需要相互学习的。

回复
Vivian
2006年11月17日 08:12 编辑
你的BLOG技术含量还挺高的啊!我是外行人,不懂!但是还是想提醒你一下,你把那么多技术性的东西写在BLOG上,一旦遇到同行人,把它都偷去了怎么办!是不是??!
 
还有啊!你为什么加我啊!给个理由!
